MediaGalleryUi/Model/SearchCriteria/CollectionProcessor/FilterProcessor/Keyword.php

```diff
@@ -67,11 +60,11 @@ private function getSelectByKeyword(string $value): Select
                 ['asset_keywords_table' => $this->connection->getTableName(self::TABLE_ASSET_KEYWORD)],
                 ['id']
             )->where('keyword = ?', $value)
-             ->joinInner(
-                 ['keywords_table' => $this->connection->getTableName(self::TABLE_KEYWORDS)],
-                 'keywords_table.keyword_id = asset_keywords_table.id',
-                 ['asset_id']
-             ),
+                ->joinInner(
```

Comment:
let's keep the consistent formatting in this method (let's move `)` to the next line)